m/configDataPreprocessingSeaexplorer.m

```diff
@@ -0,0 +1,129 @@
+%
+%  Examples:
+%    preprocessing_options = configDataPreprocessingSeaexplorer()
+%
+%  See also:
+%    PREPROCESSGLIDERDATA
+%
+%  Authors:
+%    Frederic Cyr  <Frederic.Cyr@mio.osupytheas.fr>
+%    Joan Pau Beltran  <joanpau.beltran@socib.cat>
+
+%  Copyright (C) 2013-2015
+%  ICTS SOCIB - Servei d'observacio i prediccio costaner de les Illes Balears
+%  <http://www.socib.es>
+%
+%  This program is free software: you can redistribute it and/or modify
+%  it under the terms of the GNU General Public License as published by
+%  the Free Software Foundation, either version 3 of the License, or
+%  (at your option) any later version.
+%
+%  This program is distributed in the hope that it will be useful,
+%  but WITHOUT ANY WARRANTY; without even the implied warranty of
+%  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%  GNU General Public License for more details.
+%
+%  You should have received a copy of the GNU General Public License
+%  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+  error(nargchk(0, 0, nargin, 'struct'));
+  
+  preprocessing_options = struct();
+
+  %% Navigation
+  preprocessing_options.time_list(1).time = 'Posixtime_sci';
+  preprocessing_options.position_list(1).longitude = 'NAV_LONGITUDE';
+  preprocessing_options.position_list(1).latitude  = 'NAV_LATITUDE';
+  preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  %preprocessing_options.depth_list(1).depth = 'NAV_DEPTH';
+  preprocessing_options.depth_list(1).depth = 'GPCTD_PRESSURE';
+  preprocessing_options.depth_list(2).depth = 'SBD_PRESSURE';
+  %preprocessing_options.depth_list(3).depth = 'NAV_DEPTH';
+
+
+  preprocessing_options.attitude_list(1).roll = 'Roll';
+  preprocessing_options.attitude_list(1).pitch = 'Pitch';
+  preprocessing_options.attitude_list(1).conversion = @deg2rad;
+  
+  preprocessing_options.heading_list(1).heading = 'Heading';
+  preprocessing_options.heading_list(1).conversion = @deg2rad;
+
+% $$$   preprocessing_options.time_list(2).time = 'Posixtime_nav';  
+% $$$   preprocessing_options.position_list(1).longitude = 'Lon';
+% $$$   preprocessing_options.position_list(1).latitude  = 'Lat';
+% $$$   preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  
+  %% CTD (each serie is a different configuration of glider sensors)
+  %  **All variables from a serie must be found to go forward**
+  preprocessing_options.ctd_list(1).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(1).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(1).pressure     = 'GPCTD_PRESSURE';
+  preprocessing_options.ctd_list(1).oxygen       = 'GPCTD_DOF';
+
+  preprocessing_options.ctd_list(2).conductivity = 'SBD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(2).temperature  = 'SBD_TEMPERATURE';
+  preprocessing_options.ctd_list(2).pressure     = 'SBD_PRESSURE';
+  preprocessing_options.ctd_list(2).oxygen       = 'SBD_DOF';
+
```

Comment:
I was also puzzled by the meaning of the frequency output. Yes, it is from the SBE43F, see [here](http://www.seabird.com/sbe43-dissolved-oxygen-sensor). So it is really frequency output, I guess it needs to be converted with calibration constant.

I first tried to add it on oxygen list, but as you said, some fields are required and that messed me up a bit. Then I thought doing it in the CTD list was offering minimal modifications. It also found it clearer at the end because of the name used (`GPCTD_DOF`) is similar to other GPCTD variables.

What do you think? add it with oxygen or CTD?